2.4.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+            <meta charset="UTF-8" ; />
+        <body>
+ <p><span id="demo"></span></p>
+            <script>
+          var num =[1,2,3,4,5,6];
+        document.write(num);
+
+
+            for(let i=0; i<=num.length; i++){
+                document.write("<br />"+i+"<br />");
+        if (i % 2 == 0){
```

Comment:
Find the difference between == and === and change == to use === here.

Reply:
Done